cmd/keeper/cmd/keeper.go

```diff
@@ -145,6 +146,7 @@ func init() {
 	CmdKeeper.PersistentFlags().StringVar(&cfg.pgSUUsername, "pg-su-username", user, "postgres superuser user name. Used for keeper managed instance access and pg_rewind based synchronization. It'll be created on db initialization. Defaults to the name of the effective user running stolon-keeper. Must be the same for all keepers.")
 	CmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPassword, "pg-su-password", "", "postgres superuser password. Only one of --pg-su-password or --pg-su-passwordfile must be provided. Must be the same for all keepers.")
 	CmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPasswordFile, "pg-su-passwordfile", "", "postgres superuser password file. Only one of --pg-su-password or --pg-su-passwordfile must be provided. Must be the same for all keepers)")
+	CmdKeeper.PersistentFlags().BoolVar(&cfg.replica, "replica", false, "replica only keeper")
```

Comment:
grammar nit-pick: `replica-only`

Reply:
@benwh asking for a friend :)

Do you prefer `replica-only` only for the flag or also for boolean in the keeper `replicaOnly = true`

Reply:
I was actually just pointing out the flag description (`replica only keeper` -> `replica-only keeper`) here.

I guess whether the flag should be renamed is related to the question I had above about naming - I'm still not sure on that personally, so happy to go with your preference!

Reply:
I prefer --replica-only, personally. Every keeper is otherwise potentially a replica, so it feels like this doesn't make sense otherwise?

Reply:
For the record, I'd absolutely prefer `--replica-only` over `--replica`.

Reply:
I will make the changes to be `--replica-only`